crates/bitwarden-crypto/src/lib.rs

```diff
@@ -91,6 +91,7 @@ pub use wordlist::EFF_LONG_WORD_LIST;
 mod store;
 pub use store::{KeyStore, KeyStoreContext};
 mod traits;
+mod xchacha20;
```

Comment:
This is more of an idea for the future, but it would be nice to think about moving the `aes`, `rsa` and the new `xchacha20` modules into their own separate low level algorithms module so they're not all exposed in the root.